Makefile

```diff
@@ -26,5 +26,6 @@ verify-podnetworkconnectivitychecks:
 test-e2e-encryption: GO_TEST_PACKAGES :=./test/e2e-encryption/...
 .PHONY: test-e2e-encryption
 
-test-e2e-monitoring:
 .PHONY: test-e2e-monitoring
+test-e2e-monitoring:
+	go test -v -timeout 5m ./test/e2e-monitoring/...
```

Comment:
I wasn't able to find any usages for `GO_TEST_PACKAGES` in the CI so dropped that.

Reply:
`GO_TEST_PACKAGES` is used here https://github.com/openshift/build-machinery-go/blob/5725581bdf8f467ee8fbd394f081a10df670c1ec/make/targets/golang/test-unit.mk#L12. I'd suggest using it so that we stick to the same `go test` invocation everywhere